Build failure: kime

### Steps To Reproduce

Steps to reproduce the behavior:
1. build *kime* on unstable

### Build log

```
note: /nix/store/pbixrqd120nnw8a0ygfpm3mnwlr1mbj7-binutils-2.43.1/bin/ld: cannot find -lkime_engine: No such file or directory
       >           collect2: error: ld returned 1 exit status
       >           
       >
       > error: could not compile `kime-xim` (bin "kime-xim") due to 1 previous error
       > error: linking with `/nix/store/hgw57m0jh2w10jsv8x86ng267pm5sla8-gcc-wrapper-13.3.0/bin/cc` failed: exit status: 1
```


### Notify maintainers
@Riey 

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/ki/kime/package.nix
+++ b/pkgs/by-name/ki/kime/package.nix
@@ -129,6 +129,8 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   RUST_BACKTRACE = 1;
+  # https://github.com/Riey/kime/issues/688
+  RUSTFLAGS = "-Clink-args=-L./target/release";
 
   meta = {
     homepage = "https://github.com/Riey/kime";

